### PR TITLE
Rcorp rep additions

### PR DESCRIPTION
**Pull request by Litberries**
## About The Pull Request

Adds in 5th pack for rep and locks 4th pack behind MORE research :3

## Why It's Good For The Game

I fucked up and merged this to the shi PR instead of making a new PR accidentally, so i'll just undo the shi changes 

## Changelog
:cl:

tweak: messed with rcorp rep

/:cl:

### CODE REVIEW

**Comment by Gboster-0** [comment]
eeeeeh, id say it would need some sort of a real cap
a soft-cap like not letting people kill themselfes twice on it still gives potential for very high damage

**Comment by Litberries** [comment]
> eeeeeh, id say it would need some sort of a real cap a soft-cap like not letting people kill themselfes twice on it still gives potential for very high damage

fair, but you'd still need multiple willing people to kill themselves, which basically NEVER happens during an LC13 round because teamwork is nonexistent. Not even the CLERKS will kill themselves for the greater cause these days

**Comment by InsightfulParasite** [comment]
If this had more of a effect during battle like a temporary effect then it would be good but right now it just has the tactic of someone suiciding with each knife to give it a damage buff.

**Comment by Kitsunemitsu** [comment]
No clue why this PR is so jank.
I will be taking it as it is now.

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
It's time to finally make a few changes to R-Corp Rep.

1) Scout Ravens should be available as a 4th pack option that you can spawn, to bring someone in to "spy" for you.

2) 4th Pack Progression should go Rabbits > (Berzerker) Reindeer > Rhinos.

**Comment by Litberries** [comment]
> It's time to finally make a few changes to R-Corp Rep.
> 
> 1. Scout Ravens should be available as a 4th pack option that you can spawn, to bring someone in to "spy" for you.
> 2. 4th Pack Progression should go Rabbits > (Berzerker) Reindeer > Rhinos.

Yeah sure I'll fix ravens and fuck with reindeer

File: ModularTegustation/tegu_items/representative/research/rcorp.dm [hunk 1]
```diff
@@ -53,30 +53,68 @@
...
+	research_name = "5th Pack"
+	research_desc = "We'll give you some of our shoddy 5th pack, they're not as good as the 4th pack but they're cheap."
+	cost = LOW_RESEARCH_PRICE
+	corp = R_CORP_REP
 
 // ERTs
+
+/datum/data/lc13research/mobspawner/scoutraven
+	research_name = "4th Pack Scout Raven Team"
+	research_desc = "We can send in a couple of ravens to be 'spies' for you."
```
Comment (Coxswain-Navigator) [inline ModularTegustation/tegu_items/representative/research/rcorp.dm:73]: Spies and not scouts?

**Review by Kitsunemitsu — APPROVED** [review]
Approved. Will make changes later to this, Right now it's not a problem.

File: ModularTegustation/tegu_items/representative/research/rcorp.dm (hunk 1 above)
Comment (Kitsunemitsu) [reply ModularTegustation/tegu_items/representative/research/rcorp.dm:73]: I'm going to be repurposing ravens soon to act more as eyes and ears for the rep in a future update.

**Pull request merged**